content-api-floodgate.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Floodgate
+
+[Service]
+User=content-api
+Group=content-api
+Restart=on-failure
+Environment='HOME=/home/content-api'
+Environment='JAVA_OPTS=-Xmx1024m -XX:+UseConcMarkSweepGC -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:/home/content-api/logs/gc.log -XX:ReservedCodeCacheSize=256m'
```

Comment:
It's best to set `-Xms` as well, with the same value as `-Xmx`. This ensures that the entire heap is allocated at startup, so you don't get heap resizes while the app is running.